src/models/Filter.js

```diff
@@ -231,22 +227,18 @@ export default class Filter {
       numFilters += 1;
     }
 
-    // organizationId and stakeholderUUID count as one filter
-    if (this.organizationId && this.organizationId !== ALL_ORGANIZATIONS) {
+    // if there's an organization id and it's not an array of all ids
+    if (this.organizationId && !this.organizationId.length) {
       numFilters += 1;
     }
 
     if (this.speciesId && this.speciesId !== ALL_SPECIES) {
       numFilters += 1;
     }
 
-    if (this.tokenId && this.tokenId !== 'All') {
-      numFilters += 1;
-    }
-
-    if (this.status) {
-      numFilters += 1;
-    }
+    // if (this.tokenId && this.tokenId !== 'All') {
```

Comment:
What's the reason for commenting this out?

Reply:
Mental blip.  I was only considering Verify where we aren't currently filtering by token.